lib/ansible/modules/cloud/docker/docker_swarm.py

```diff
@@ -427,11 +437,29 @@ def inspect_swarm(self):
             data = self.client.inspect_swarm()
             json_str = json.dumps(data, ensure_ascii=False)
             self.swarm_info = json.loads(json_str)
+
             self.results['changed'] = False
             self.results['swarm_facts'] = self.swarm_info
+
+            unlock_key = self.get_unlock_key()
+            self.swarm_info.update(unlock_key)
         except APIError:
             return
 
+    def get_unlock_key(self):
```

Comment:
I think this should be in `AnsibleDockerSwarmClient` class

Reply:
I disagree. This is only needed for `docker_swarm`, not for `docker_swarm_info`.

Reply:
I too disagree. It's very `docker_swarm` specific as it tests if the module-parameter `autolock_managers` has changed. The default handling of returning the raw `UnlockKey` key can be argued to be lifted into `AnsibleDockerSwarmClient` as discussed in https://github.com/ansible/ansible/pull/54490#discussion_r270600286. But I do think we need a specific method here to do the diff-checking.